### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by ashatskyy**
[DEMO LINK](https://ashatskyy.github.io/react_person/)

### CODE REVIEW

File: src/components/Person/Person.jsx [hunk 1]
```diff
@@ -1 +1,11 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    {'age' in person ? <p className="Person__age">I am {person.age}</p> : null}
+    <p className="Person__partner">
+      {person.isMarried === true
```
Comment (etojeDenys) [inline src/components/Person/Person.jsx:6]:
```suggestion
      {person.isMarried
```

File: src/components/Person/Person.jsx [hunk 2]
```diff
@@ -1 +1,11 @@
...
+        ? `${person.partnerName} is my ${person.sex === 'm' ? `wife` : `husband`}`
+        : `I am not married`}
```
Comment (etojeDenys) [inline src/components/Person/Person.jsx:8]: nested ternary operators are a bad practice


decide before jsx what text will be rendered and just render in jsx. something like that

```
const personPartnerType = sex === 'm' ? 'wife' : 'husband';
const personPartner = isMarried ? `${partnerName} is my ${personPartnerType}` : 'I am not married'
```

**Review by etojeDenys — CHANGES_REQUESTED** [review]

File: src/components/Person/Person.jsx [hunk 3]
```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const personPartnerType = person.sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {person.name}</h2>
```
Comment (volodymyr-soltys97) [inline src/components/Person/Person.jsx:6]: Use destructuring for `person`

File: src/components/Person/Person.jsx [hunk 4]
```diff
@@ -1 +1,17 @@
...
+      {'age' in person ? (
+        <p className="Person__age">I am {person.age}</p>
+      ) : null}
```
Comment (volodymyr-soltys97) [inline src/components/Person/Person.jsx:9]: Fix it after destructuring
```suggestion
      {age && (
        <p className="Person__age">I am {age}</p>
      )}
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
GJ!
Let's make your code better

File: src/components/Person/Person.jsx (hunk 3 above)
Comment (ashatskyy) [reply src/components/Person/Person.jsx:6]: Good evening! In this task instructions we see condition: ''pass the whole person as a prop <Person person={misha} /> (not individual fields)'' Is your comment matches this condition?

**Pull request closed without merging**